README.md

```diff
@@ -1,91 +1,63 @@
 
+---
 
-**If you are not yet a user of IDS GitHub**, please join us by clicking on the button below and submitting a request to participate. *Requests are typically processed within two business days and in most cases, invitations are sent within several hours.*
+## 🧭 Purpose of This Repository
 
-[![Join](/images/button_click-to-join-ids-githubv2.png.png)](https://forms.office.com/r/LMFt6pbji4)
+This repository acts as the **main entry point** for all IDSA activities on GitHub.  
+It connects you to the latest frameworks, specifications, and guidelines shaping data spaces.
 
+If you are new to the IDSA GitHub, [**click here to join**](https://forms.office.com/pages/responsepage.aspx?id=NNZGs_usx0K9RPFVfuibGz1qA21VHyZEj6dyjHL7iBdUM0ZVMzlEMkJTUlhOSEVEWEQyMjZPOUNYTi4u&route=shorturl).
 
-# Table of contents
+---
 
--   What is Available Here?
-    -   [IDS Reference Architecture Model (IDS RAM)](#IDS-RAM)
-    -   [IDSA Rulebook](#IDSA-RULEBOOK)
-    -   [IDS Reference Testbed](#IDS-Reference-Testbed)
-    -   [IDS Knowledge Base](#Knowledge-Base)
--   [IDSA at a Glance](#IDSA-at-a-Glance)
--   [How to Contribute?](#How-to-Contribute?)
--   [Management of Releases & License](#Management-of-Releases-&-License)
+## 📚 The IDSA Value Chain of Documents
 
+The IDSA documentation follows a clear value chain — from **vision to implementation**.  
+Each layer builds on the previous one to ensure consistency, quality, and usability.
 
-## :blue_book: IDS Reference Architecture Model (IDS RAM) <a name="IDS-RAM"></a>
-The [IDS RAM](https://github.com/International-Data-Spaces-Association/RAM5) is the beating heart of the IDS. It is **accessible to IDSA members** and includes the standards for secure and sovereign data exchange, certification and governance. The RAM serves as the blueprint for trusted ecosystems for data exchange and processing.
+| Level | Description | Repository / Resource |
+|-------|--------------|-----------------------|
+| **Vision & Principles** | Strategic foundation for data sovereignty, trust, and interoperability. | [Manifesto for IDS](https://docs.internationaldataspaces.org/ids-knowledgebase/manifesto-for-international-dataspaces/), [Most Important Documents](https://internationaldataspaces.org/publications/most-important-documents/) |
+| **Architecture & Standards** | The **IDS Reference Architecture Model (RAM)** defines the conceptual and technical blueprint of trusted data spaces. | [IDS RAM 5 (Members-only)](https://github.com/International-Data-Spaces-Association/RAM5) <br> [Previous version: RAM 4.0](https://github.com/International-Data-Spaces-Association/IDS-RAM_4_0) |
+| **Governance & Compliance** | The **IDSA Rulebook** provides functional, technical, operational, and legal guidelines for building and managing data spaces. | [IDSA Rulebook](https://github.com/International-Data-Spaces-Association/IDSA-Rulebook) |
```

Comment:
I'd switch Rulebook and Architecture document(s) to reflect the "value chain" of documents.